src/rmp/src/Restructure.cpp

```diff
@@ -124,7 +127,7 @@ void Restructure::getBlob(unsigned max_depth)
 
   getEndPoints(ends, opt_mode_ <= Mode::AREA_3, max_depth);
   if (ends.size()) {
-    sta::PinSet fanin_fanouts = resizer_->findFaninFanouts(&ends);
+    sta::PinSet fanin_fanouts = opt_mode_ > Mode::AREA_3 ? resizer_->findFanins(&ends) : resizer_->findFaninFanouts(&ends);
```

Comment:
it is confusing to call this fanin_fanouts when it may just be fanins.
Use helper for mode.

Reply:
Changed variable name.